app/controllers/application_controller.rb

```diff
@@ -90,10 +91,4 @@ def prompt_to_set_up_2fa
   def prompt_to_enter_otp
     redirect_to user_two_factor_authentication_url
   end
-
-  def track_get_requests
```

Comment:
IIUC, the point of this method was to avoid logging POST requests, since those may contain PII or other sensitive info. My brief test shows that yes, POST requests are being logged since we are using the standard Rails logging system.

I'm not sure how this would be configured differently in production to avoid logging sensitive info. Can you comment?